hopsworks-common/src/main/java/io/hops/hopsworks/common/dao/pythonDeps/PythonDepsFacade.java

```diff
@@ -82,7 +82,32 @@
 
   private final static Logger logger = Logger.getLogger(PythonDepsFacade.class.
       getName());
-
+  
+  //User upgradable libraries we installed for them
+  public static final ArrayList<String> providedLibraryNames = new ArrayList<String>() {
```

Comment:
@kouzant could we move these to the variables table, so we don't have to redeploy the app to change the list? Unless there is a UI way to modify it (I don't think so but I don't remember for sure).

Reply:
I'll amend this PR and hopsworks-chef